src/hooks/queries/useGetEvent.ts

```
@@ -1,5 +1,5 @@
-import { useQuery } from "@tanstack/react-query";
 import { eventApi } from "@/apis/eventApi";
+import { useQuery } from "@tanstack/react-query";
```

ESLint

[error] `@tanstack/react-query` import should occur before import of `@/apis/eventApi`

(import/order)